SpriterDotNet/SpriterParser.cs

```diff
@@ -13,6 +13,7 @@ namespace SpriterDotNet
     public static class SpriterParser
     {
         public static readonly ICollection<ISpriterParser> Parsers = new List<ISpriterParser>();
+        internal static Action<Spriter> InitAction = parser => { Init(parser); };
```

Comment:
Actually, did some reading. Rethinking this. I'm probably being overzealous here for diminishing returns.